x-pack/solutions/observability/plugins/apm/server/routes/profiling/route.ts

```diff
@@ -47,20 +46,30 @@ const servicesFlamegraphRoute = createApmServerRoute({
       const { start, end, kuery, transactionName, transactionType, environment } = params.query;
 
       const indices = apmEventClient.getIndicesFromProcessorEvent(ProcessorEvent.transaction);
+      const stacktraceIdsField = await getStacktracesIdsField({
+        apmEventClient,
+        start,
+        end,
+        environment,
+        serviceName,
+        transactionType,
+        transactionName,
+        kuery,
+      });
 
       return fetchFlamegraph({
         profilingDataAccessStart,
         core,
         esClient: esClient.asCurrentUser,
-        start,
-        end,
+        start: start / 1000,
```

Comment:
Just wondering - why is this `/1000` needed here?

Asking because I ran into off by 1000 issues across old APM data vs. OTel data.

Reply:
That's because Profiling data is fetched using seconds and not milliseconds.